scripts/generator/graphql_generator/csharp/Editor/ShopifyOnboardingPanel.cs.erb

```diff
@@ -0,0 +1,114 @@
+﻿using System.Net;
+
+namespace <%= namespace %>.SDK {
+    using System.Linq;
+    using UnityEditor;
+    using UnityEngine;
+
+    public class ShopifyOnboardingPanel : EditorWindow {
+        private const string HasSeenShopifyBuyPopupEditorPrefsKey = "has_seen_shopify_buy_popup";
+
+        [InitializeOnLoadMethod]
+        public static void OnLoad() {
+            if (PopupHasBeenShown()) return;
+            ShowWindow();
+        }
+
+        [MenuItem("Shopify/Help")]
+        public static void ShowWindow() {
+            var window = EditorWindow.GetWindow<ShopifyOnboardingPanel>();
+            window.ShowPopup();
+
+            EditorPrefs.SetBool(HasSeenShopifyBuyPopupEditorPrefsKey, true);
+        }
+
+        private static bool PopupHasBeenShown() {
```

Comment:
nit: If this is returning a boolean, can we name it similarly to `HasSeenShopifyBuyPopupEditorPrefsKey`
```
if (HasSeenPopup()) return;
```